### ISSUE TITLE
Normalization of input

### ISSUE THREAD

**Issue by LeoniekevandenBulk** [facebookresearch/VMZ#60]
I am trying to use the pretrained models in pytorch and am therefore writing a replacement for the VideoInput layer, but I am not sure on the exact normalization that was used on the input. And is it scaled between -1 and 1 or 0 and 1 at the end? It would be very helpful if this could be elaborated on.

Thanks!

**Comment by keunhong** [comment]
I've been digging through the code and can't seem to find a clear answer for this. How are the inputs normalized? What mean/std are used? Are they scaled between -1 and 1?

**Comment by dutran** [comment]
we normalize it to zero mean and unit variance. Here are the mean/std https://github.com/pytorch/pytorch/blob/master/caffe2/video/video_input_op.h#L374-L378
